README.md

```diff
@@ -21,9 +21,12 @@
 
 **🛑 紧急通知：目前发现有黑客利用 `lencx/ChatGPT` 的热度，在 fork 项目后植入木马，重新构建安装程序。如果你身边有朋友正在使用此桌面应用，请提醒 TA 们不要随意下载不明链接。现在项目将删除其他安装途径，仅提供此下载链接 https://github.com/lencx/ChatGPT/releases**
 
+**🛑 अत्यावश्यक सूचना: फोर्क परियोजना के बाद ट्रोजन हॉर्स लगाने और इंस्टॉलर को फिर से बनाने के लिए एक हैकर ने `lencx/ChatGPT` की गर्मी का फायदा उठाते हुए पाया गया है। यदि आपके आस-पास ऐसे मित्र हैं जो इस डेस्कटॉप एप्लिकेशन का उपयोग कर रहे हैं, तो कृपया उन्हें याद दिलाएं कि अज्ञात लिंक को स्वतंत्र रूप से डाउनलोड न करें। अब परियोजना अन्य स्थापना तरीकों को हटा देगी और केवल यह डाउनलोड लिंक प्रदान करेगी https://github.com/lencx/ChatGPT/releases**
+
+
 ---
 
-**It is an unofficial project intended for personal learning and research purposes only. During the time that the ChatGPT desktop application was open-sourced, it received a lot of attention, and I would like to thank everyone for their support. However, as things have developed, there are two issues that seriously affect the project's next development plan:**
+**This is an unofficial project intended for personal learning and research purposes only. Since, when ChatGPT desktop application was open-sourced, it received a lot of attention, and I would like to thank everyone for their support. However, as things have developed, there are two issues that seriously affect the project's next development plan:**
```

Comment:
The English is not sound here. `Since, when...` don't make good English sense. Rather say `Since ChatGPT Desktop Application was made open-source...`

Reply:
Thank you for your review sir